assembl/static2/css/components/harvesting.scss

```diff
@@ -0,0 +1,117 @@
+.left-side-harvesting-button {
+  &:hover {
+    .assembl-icon-catch:before {
+      font-weight: bold;
+    }
+
+    .left-side-harvesting-button__label {
+      visibility: visible;
+    }
+
+    .left-side-harvesting-button__button__inside {
+      background-color: #ffed5b;
+    }
+  }
+
+  .left-side-harvesting-button__label {
+    position: absolute;
+    left: -100px;
+    top: 0;
+    width: 100px;
+    margin-top: 11px;
+    padding-right: 9px;
+    text-align: right;
+    visibility: hidden;
+  }
+
+  .left-side-harvesting-button__button {
+    background: white;
+    border: 1px solid #ccc;
+    border-radius: 50%;
+    width: 44px;
+    height: 44px;
+    box-shadow: 2px 2px 4px 0 #ccc;
+
+    .left-side-harvesting-button__button__inside {
+      text-align: center;
+      margin: 4px;
+      width: calc(100% - 8px);
+      height: calc(100% - 8px);
+      background-color: #fff7b5;
+      border-radius: 50%;
+      display: flex;
+    }
+  }
+}
+
+.confirm-harvest-button {
+  text-align: center;
+  margin: 4px;
+  padding: 4px 0;
+}
+
+.extracts.extracts--is-harvesting {
+  left: 0;
+  max-width: 355px;
+  width: 20vw;
+  line-height: initial;
+
+  .extract {
+    border-left: $border-theme;
+    border-bottom: 1px solid #ddd;
+    border-right: 1px solid #ddd;
+    border-top: 1px solid #ddd;
+    background-color: $white;
+
+    .extract-txt {
+      quotes: "«" "»";
```

Comment:
This type of quoting only applies to the French language. It does not exist in English. So, perhaps this should be set in JS after checking locale?

Reply:
This is how Roxana's mockups look like. I'm not aware of a requirement of showing this styling differently depending on the language.